README.md

```diff
@@ -94,9 +94,10 @@ The following is an example.
 ```access transformers
 messagingProtocols=rocketmq
 protocolHandlerDirectory=./protocols
+loadManagerClassName=org.apache.pulsar.broker.loadbalance.impl.ModularLoadManagerImpl
 ```
 
-2. Set RocketMQ service listeners
+1. Set RocketMQ service listeners

```

Comment:
It seems that the `2` here does not need to be changed to `1`

Reply:
Sorry, it seems that it's changed automatically by the vscode linter. I will revert this change in next commit.